### PR TITLE
fix bug #1147: Missing Crypt32.lib when linking OpenSSL statically

### PR DESCRIPTION
**Pull request by 23W**
fix for bug #1147 
I add the crypt32 library to dependency list in any case when using OpenSSL in Windows.

### CODE REVIEW

**Comment by 23W** [comment]
I can't understand why ` continuous-integration/appveyor/pr` is broken? Logs with errors are empty.

**Comment by Lekensteyn** [comment]
Logs say:

> Build execution time has reached the maximum allowed time for your plan (60 minutes).

Maybe there were network issues.

Also I do not think this patch is correct, the code path is executed if "NOT UNIX" is false, i.e. if "UNIX" is true. Unless I am misreading stuff

**Comment by 23W** [comment]
@Lekensteyn 
No, it is works `if not CURL_WINDOWS_SSPI` in block `if not UNIX`, i.e. if assembly for Windows when SSPI (WinSSL) is not enabled. So, I am checking if OPENSSL is enabled and adding crypt32 to the dependency list.

File: CMakeLists.txt [hunk 1]
```diff
@@ -590,6 +590,8 @@ if(NOT UNIX)
         set(CURL_LIBS ${CURL_LIBS} "crypt32")
       endif()
     endif()
+  elseif(USE_OPENSSL)
+        set(CURL_LIBS ${CURL_LIBS} "crypt32")
```
Comment (Lekensteyn) [inline CMakeLists.txt]: Please fix this misleading indentation (use two like the others)

**Comment by Lekensteyn** [comment]
@23W Oh, I was mislead by the indentation. If you update the commit and push again, it will trigger the CI again.

**Comment by bagder** [comment]
Thanks, both of you!

**Pull request closed without merging**